misc/python/materialize/benchmark_utils.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""Benchmark utilities."""
+
+from materialize import buildkite
+
+
+def resolve_tag_of_common_ancestor() -> str:
+    if not buildkite.is_in_pull_request():
+        print("Not in a pull request, resolved common-ancestor to latest")
```

Comment:
This decision was reached in your Nightly run, even though it is in a pull request :-(

```
_bk;t=1697295648818Not in a pull request, resolved common-ancestor to latest
```

Reply:
I will use the following logic instead of relying on `$BUILDKITE_PULL_REQUEST`:
`if $BUILDKITE_BRANCH not equal to $BUILDKITE_PIPELINE_DEFAULT_BRANCH`